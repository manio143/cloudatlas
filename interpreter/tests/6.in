SELECT min(sum(distinct(2 * level)) + 38 * size(contacts)) AS sth WHERE num_cores < 8
