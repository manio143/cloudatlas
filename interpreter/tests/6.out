/uw: sth: 80
/pjwstk: sth: 80
/: sth: NULL
